Source/FolioReaderWebView.swift

```diff
@@ -346,10 +382,12 @@ open class FolioReaderWebView: UIWebView {
     
     // MARK: - Java Script Bridge
     
-    @discardableResult open func js(_ script: String) -> String? {
-        let callback = self.stringByEvaluatingJavaScript(from: script)
-        if callback!.isEmpty { return nil }
-        return callback
+    open func js(_ script: String, completion: @escaping JSCallback) {
+        
+        self.evaluateJavaScript(script) { (result, error) in
+            completion(result as? String)
+        }
+
```

Comment:
Making the completion optional will allow you to avoid having to implement it.

Like this`.js("resetCurrentSentenceIndex()")` instead of `.js("resetCurrentSentenceIndex()") { _ in }`
```suggestion
    open func js(_ script: String, completion: JSCallback? = nil) {
        evaluateJavaScript(script) { (result, error) in
            completion?(result as? String)
        }
    }
```

Reply:
sorry can you help me. i add  js(_ script: String, completion: JSCallback? = nil) in FolioReaderWebView
<img width="954" alt="Screen Shot 2563-07-22 at 10 40 25" src="https://user-images.githubusercontent.com/65211118/88131729-d5154600-cc07-11ea-8efa-288f29b03a0c.png">